test/fixtures/v5/end_of_road/sharp_right_destination.json

```diff
@@ -10,7 +10,7 @@
     },
     "instructions": {
         "ar": "انعطف يمين حادا نحو Destination 1",
-        "da": "Drej til skarpt højresving mof Destination 1",
+        "da": "Drej til skarpt højresving mod Destination 1",
```

Comment:
Should be "Drej skarpt til højre mod Destination 1"

Reply:
The same question as for sharp left.